docs/quick-start.md

```diff
@@ -7,7 +7,15 @@ In this page, we will guide you through the steps to:
 
 ## Before you begin: prerequisites
 
-To try Numaflow, you will first need to create a local cluster. This can be done simply using [Docker Desktop](https://www.docker.com/blog/how-kubernetes-works-under-the-hood-with-docker-desktop). Follow the steps on [Docker's documentation](https://docs.docker.com/desktop/kubernetes/) to learn how to create a local Kubernetes cluster.
+To try Numaflow, you will first need to setup using one of the following options to run container images:
+- [Docker Desktop](https://www.docker.com/blog/how-kubernetes-works-under-the-hood-with-docker-desktop)
```

Comment:
Using the link below for downloading might be better.

https://docs.docker.com/get-docker/

Reply:
updated